### DISCUSSION TITLE
Video URL not found

### DISCUSSION THREAD

**Discussion by Tjfireball07** [pystardust/ani-cli#662] [General] [answered]
My ani-cli keeps on saying 

Loading episode 1... 
base64: invalid input
bad decrypt
140679626290560:error:0606506D:digital envelope routines:EVP_DecryptFinal_ex:wrong final block length:../crypto/evp/evp_enc.c:599:
Video URL not found

is there any fix for this it just started happening i am using Kali if that is any help i have looked for a while and can not find any fix that works for me and i have reinstalled it 

![Ani-cli](https://user-images.githubusercontent.com/99737850/163701058-75424362-78e8-4222-abbe-c5af86c19dd0.png)

**Comment by port19x** [comment, marked as answer]
Yes, this already happened and got fixed a bunch of times the past few days.
It's cryptography, I'd have been surprised if you found the new keys yourself
